docs/content/docs/test/TestDataClearing.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Test Data Clearing
+type: docs
+---
+
+# Test Data Clearing
+
+Every test runs in clear application state without any database files and SharedPreferences.
+
+## Running locally
+
+When a test is running locally all produced data is cleared on test finish with ADB shell command `pm clear`.
+You can control this behaviour of Android Test Orchestrator with `clearPackageData` flag like the following:
```

Comment:
It's important to declare that it works only in ANDROIDX_TEST_ORCHESTRATOR.
And we don't want to use it anymore.

Reply:
I removed this document completely